.github/workflows/detect_pull_request_preview.yml

```diff
@@ -0,0 +1,24 @@
+name: Detect Pull Request Preview
```

Comment:
Can you name this more in line with documentation.yml and manifest.yml? These strings show up in GitHub UI with limited width, so they need to be pretty short. If it's possible to share the same name for two workflows, calling both "pr-preview" and just having the jobs named differently would be ideal. That might be an error though.

Reply:
Sure